main.go

```diff
@@ -107,6 +107,7 @@ func Init() {
 }
 
 func versionInfo() map[string]interface{} {
+	stats.NewTaggedStat("rudder_server_config", stats.GaugeType, stats.Tags{"version": version, "major": major, "minor": minor, "patch": patch, "commit": commit, "buildDate": buildDate, "builtBy": builtBy, "gitUrl": gitURL, "TransformerVersion": transformer.GetVersion(), "DatabricksVersion": misc.GetDatabricksVersion()}).Gauge(1)
```

Comment:
please make sure we publish this stat only once, during startup

Reply:
moved it to `Run()` which is called only once.